client/gulpfile.babel.js

```diff
@@ -41,13 +41,14 @@ const paths = {
   ],
   styles: [
     'src/**/*.scss'
-  ]
+  ],
+  dest: '../molo/core/static'
```

Comment:
Need to put them there so that the scaffolded out app can find it, happy to put somewhere else though.